src/fiatExchanges/SelectProvider.tsx

```diff
@@ -195,7 +200,7 @@ export default function SelectProviderScreen({ route, navigation }: Props) {
     !coinbaseProvider.restricted &&
     coinbasePayEnabled &&
     appId &&
-    digitalAsset === CiCoCurrency.CELO
+    tokenInfo.symbol === celoTokenSymbol
```

Comment:
I'm not sure if using `celoTokenSymbol` is better than the string literal 'CELO', since Celo has historically had two different symbols, and it's not clear which one is meant by `celoTokenSymbol`

Reply:
It's a little more opaque than `CiCoCurrency.CELO` since our convention with enums is to name the key the same as the value, so you know that `SomeEnum.Foo` is going to be the same as `"Foo"` (not that I think we should go back to using CiCoCurrency-- we need to deprecate that enum since it no longer serves a purpose)